### PR TITLE
Update from main

### PR DESCRIPTION
**Pull request by charlotte-avery**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
     url="https://github.com/centrefornetzero/smart-building-rating-calculator",
-    packages=find_packages("src", package_dir={"": "src"}),
+    package_dir={"": "src"},
```
Comment (shengy90) [inline setup.py:12]: Why are you removing `package_dir={"": "src"}`?

Comment (charlotte-avery) [reply setup.py:12]: Replacing `packages=find_packages()` with `package_dir={"": "src"}` as a bug fix (error message stated that find_packages has no `package_dir` key word)

Comment (shengy90) [reply setup.py:12]: You should also directly pin requirements in your setup.py like so:

```python
install_requires = (
    "pandas==2.0.2",
    "numpy==1.25.0",
)

setup(
    package_dir={"": "src"},
    packages=find_packages("src"),
    install_requires=install_requires,
```

**Review by shengy90 — CHANGES_REQUESTED** [review]
Pin requirements explicitly. Also why are we removing package_dir={"": "src"}?

File: setup.py (hunk 1 above)
Comment (shengy90) [reply setup.py:12]: >Replacing packages=find_packages() with package_dir={"": "src"} as a bug fix (error message stated that find_packages has no package_dir key word)

I think that's cause it's supposed to be in a separate argument!

**Pull request merged**